facades/PC/build.gradle

```diff
@@ -81,11 +81,33 @@ task game(type:JavaExec) {
     classpath project(':engine').configurations.runtime
 }
 
+task debug(type:JavaExec) {
```

Comment:
Is this copy & paste from the `game` task? If so, maybe you can extend the existing task or derive both from a common ancestor to avoid code duplication.

I'm somewhat skeptical that this is a realistic use case, but I see potential for the `runServer` task. It would be good if JMX access could be exposed that easily.

I haven't checked, but I think that `server` has become obsolete, you might not need it. Also, the port `1044` should be communicated somehow, e.g. through logging? It would be nice if the `suspend` flag was (optionally) configurable to enable debugging the initialization process.

Apologies for the extra wishes :smile: